README.md

```diff
@@ -35,16 +51,15 @@ Laminar was designed to be used in the [Amethyst][amethyst] game engine and is l
 ## Features
 These are the features this crate provides:
 
-- UDP-based protocol
-- Connection tracking
+- UDP-based Protocol
+- Connection Tracking
 - Automatic Fragmentation
-- Unreliable and Reliable packets
-- Protocol versioning
-- RTT estimation
-- Link conditioner to simulate packet loss and latency
-- Well tested by integration tests and unit tests
-- Good error handling
-- Benchmarks
+- Packets Types: Unreliable: Sequenced, Reliable: Unordered, Sequenced, Ordered.
```

Comment:
```suggestion
- Packets Types: Unreliable and Sequenced, Reliable and Unordered, Sequenced and Ordered.
```

Reply:
Now we cant have: unreliable only, or unreliable sequenced. This is not correct we have the following types:

Reliable + Ordered
Reliable + Unordered
Reliable + Sequenced
Unreliable
Unreliable + Sequenced.